.github/workflows/main.yaml

```diff
@@ -1,4 +1,14 @@
-on: [push, pull_request]
+on:
```

Comment:
This fixes the issue where we were triggering a push and pr CI jobs. If you want CI for another branch, create a PR.

https://github.community/t/how-to-trigger-an-action-on-push-or-pull-request-but-not-both/16662/14

Reply:
nice catch - thanks